packages/formbricks-api-wrapper/src/utils/makeRequest.ts

```diff
@@ -0,0 +1,37 @@
+import fetch from "node-fetch";
```

Comment:
In our repository the package is mainly used client side (formbricks-js) but should also work server side in. What do we need to do to make it work both client and server side?

Reply:
I added the `node-fetch` for that purpose itself! For non-browser interactions, to have the `fetch`, we can use the node-fetch packale